db.json

```diff
@@ -6486,7 +7622,7 @@
   "application/xhtml+xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["xhtml","xht"]
+    "extensions": ["xhtml","xht","html","htm"]
   },
```

Comment:
https://www.iana.org/assignments/media-types/application/xhtml+xml lists only "xhtml","xht"

This is one where I feel that adding `"html","htm"` probably makes this mime-db less useful, as those file extensions are very commonly used, and very unlikely to be XML most of the time.